plone/app/z3cform/templates/select_input.pt

```diff
@@ -1,63 +1,35 @@
 <div xmlns="http://www.w3.org/1999/xhtml"
      xmlns:tal="http://xml.zope.org/namespaces/tal"
      tal:omit-tag=""
->
-  <select class=""
-          id=""
-          name=""
-          size=""
-          tabindex=""
+><!--! NOTE: since this template is also used for Select2Widget
+       we have to make sure that the ":list" isn't set in that case.
+       "pat-select2" refactors the input value for multiple selects
+       to separator concated text input.
+       This behavior hopefully will be gone soon when "pat-select2"
+       is upgraded to Select2 >= 4.x -->
```

Comment:
```suggestion
    ><tal:c condition="nothing">
    NOTE:
    Since this template is also used for Select2Widget we have to make sure
    that the ":list" isn't set in that case. `pat-select2` refactors the input
    value for multiple selects to separator concated text input. This behavior
    hopefully will be gone soon when `pat-select2` is upgraded to Select2 >=
    4.x
    </tal:c>
```